qiskit_ibm_runtime/runtime_job.py

```diff
@@ -240,67 +239,6 @@ def submit(self) -> None:
             "QiskitRuntimeService.run() to submit a job."
         )
 
-    def queue_position(self, refresh: bool = False) -> Optional[int]:
```

Comment:
`RuntimeJob` is a publicly documented class (https://quantum.cloud.ibm.com/docs/en/api/qiskit-ibm-runtime/runtime-job), which means, we can't remove this method without prior deprecation. I suggest including the deprecation warning as part of #2296.

Reply:
Done in [c588863](https://github.com/Qiskit/qiskit-ibm-runtime/pull/2296/commits/c58886313a9949f4dd9c01dc0e6392342f1d8114)